setup.py

```diff
@@ -94,12 +93,12 @@ def get_extra_compile_args():
 
 def get_extra_link_args():
     if sys.platform == 'win32':
-        args = ['/subsystem:windows', '/machine:I386']
+        args = ['/subsystem:windows', '/machine:x64']
```

Comment:
agreed!

On Mar 1, 2015, at 2:07 PM, "Didrik Pinte" <notifications@github.com<mailto:notifications@github.com>> wrote:


In setup.py<https://github.com/tnicolas/pyql/pull/1#discussion_r25570076>:

> @@ -94,12 +93,12 @@ def get_extra_compile_args():
>
>  def get_extra_link_args():
>      if sys.platform == 'win32':
> -        args = ['/subsystem:windows', '/machine:I386']
> +        args = ['/subsystem:windows', '/machine:x64']


we should add an auto-detection of the bitness of the platform in place of having to change it by hand.

-
Reply to this email directly or view it on GitHub<https://github.com/tnicolas/pyql/pull/1/files#r25570076>.